src/wasm-type.h

```diff
@@ -21,21 +21,58 @@
 
 namespace wasm {
 
-enum Type {
-  none,
-  i32,
-  i64,
-  f32,
-  f64,
-  v128,
-  anyref,
-  exnref,
-  // none means no type, e.g. a block can have no return type. but unreachable
-  // is different, as it can be "ignored" when doing type checking across
-  // branches
-  unreachable
+class Type {
+  uint32_t id;
```

Comment:
why not make `id` a `ValueType`?

Reply:
For multivalue it will need to be a general integer and I didn't see much benefit of using `ValueType` in the short term.